concordium-sdk/src/main/java/com/concordium/sdk/responses/blockinfo/BlockInfo.java

```diff
@@ -1,90 +1,89 @@
 package com.concordium.sdk.responses.blockinfo;
 
+import com.concordium.sdk.responses.BlockIdentifier;
 import com.concordium.sdk.serializing.JsonMapper;
 import com.concordium.sdk.transactions.Hash;
 import com.fasterxml.jackson.core.JsonProcessingException;
-import lombok.*;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
 import java.time.OffsetDateTime;
 
 @ToString
-@Data
-@RequiredArgsConstructor
 @Getter
-public class BlockInfo {
-    /**
-     *  Hash of the block.
-     */
-    private Hash blockHash;
+@SuperBuilder
+@Jacksonized
+public class BlockInfo extends BlockIdentifier {
```

Comment:
I think it should simply be a field on the `BlockInfo`, and then you manually create the two `getBlockHash` and `getBlockHeight` functions such that the API becomes backwards compatible. 

Whether these two fields are kept as two fields on inside the `BlockIdentifier` should be an implementation detail.

Reply:
This has been fixed by making `BlockIdentifier` a field in `BlockInfo`